scripts/install/install_macos.sh

```diff
@@ -272,6 +278,36 @@ check_prereqs()
   decrease_indent
 }
 
+# Test connection to BindPlane if it was specified
+connection_check()
+{
+  info "Help Me Obi-Wan Kenobi!"
+  if [ $check_bp_url == "true" ] ; then
+    if [ -n "$opamp_endpoint" ]; then
+      HTTP_ENDPOINT="$(printf "${opamp_endpoint}" | sed 's#ws#http#' | sed 's#/v1/opamp$##')"
+      info "Testing connection to BindPlane: $fg_magenta$HTTP_ENDPOINT$reset..."
+
+      if curl -s ${HTTP_ENDPOINT} > /dev/null; then
+        succeeded
+      else
+        failed
+        warn "Connection to BindPlane has failed. Do you wish to continue installation?"
+        increase_indent
+        printf "${fg_yellow}Do you wish to continue installation?${reset}  "
```

Comment:
shellcheck is hitting me with this for this line:
https://www.shellcheck.net/wiki/SC2059

Maybe just needs an ignore comment like elsewhere in the script

Reply:
I forgot to run shellcheck after making some "changes" before commiting. Thanks for the reminder.